.github/workflows/dusk-tests.yml

```
@@ -0,0 +1,100 @@
+name: Dusk Tests
+
+on:
+  push:
+    branches: [ main, master ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)